docs/prod-host-setup.md

```diff
@@ -46,6 +46,21 @@ for consuming and storing this data safely. We suggest using any upper-bounded
 forms of storage, such as fixed-size or ring buffers, programs like `journald`
 or `logrotate`, or redirecting to a named pipe.
 
+### Logging and performance
+
+There is extra latency introduced in Firecracker functionality as the process
+will log various error or warning messages. Some host configurations can have
+an effect on the performance of the process as Firecracker still has to use
+the host devices, such as the serial device, to write the logs generated
```

Comment:
I think this should be rephrased. Firecracker doesn't use the serial device. Host kernel logs are generated during normal Firecracker operations. And these logs can end up on the serial console depending on the host config.